migration/src/main/java/org/corfudb/migration/LayoutReset.java

```diff
@@ -0,0 +1,61 @@
+package org.corfudb.migration;
+
+import static com.google.common.collect.Sets.newHashSet;
+
+import java.io.File;
+import java.util.Set;
+
+/**
+ * Layout Reset.
+ * Resets the layout and epochs. This allows the node to be bootstrapped with a fresh layout.
+ * This is valid for version 0.3.0.
+ */
+public class LayoutReset {
+
+    /**
+     * Resets all layouts and epoch related metadata.
+     *
+     * @param args Takes the CorfuDB directory.
+     */
+    public static void main(String[] args) {
+        if (args.length < 1) {
+            throw new IllegalArgumentException("Expected parameters: CorfuDB data directory");
+        }
+        resetLayoutDatastore(args[0]);
+    }
+
+    /**
+     * Resets all layout and epoch related metadata files.
+     *
+     * @param dirStr Directory path for CorfuDB.
+     */
+    public static void resetLayoutDatastore(String dirStr) {
+
+        Set<String> dsKeyPrefixes = newHashSet("SERVER_EPOCH",
+                "LAYOUT",
+                "PHASE_1",
+                "PHASE_2",
+                "LAYOUTS",
+                "MANAGEMENT",
+                "SEQUENCER_EPOCH",
+                "LOGUNIT_EPOCH_WATER_MARK");
+
+        File dir = new File(dirStr);
+        File[] foundFiles = dir.listFiles((dir1, name) -> {
+            for (String prefix : dsKeyPrefixes) {
+                if (name.startsWith(prefix)) {
```

Comment:
It simplifies if we just return name.startsWith(prefix)

Reply:
Done.